### ISSUE TITLE
Implement an architecture with CNN and LSTM

### ISSUE THREAD

**Issue by kapiblue** [JankowskiDaniel/Neural-Deep-Retina#15]

**Comment by JankowskiDaniel** [comment]
The first experiment shows that the model can overfit well on the training set, however, at the same time is lost in the testing data.
**Training**
![outputs_and_targets](https://github.com/user-attachments/assets/558b8af2-40e3-4206-8b8c-580d4be32ad6)
**Testing**
![outputs_and_targets_test](https://github.com/user-attachments/assets/48970ecc-42b3-492e-b1ea-db73ce7d2925)

**Comment by JankowskiDaniel** [comment]
A few general observations:

* **StandardScaler()** works better than **MinMaxScaler()**,
* **MinMaxScaler()** (range [0,1]) was tested also with **Sigmoid()** on the output, but something was not working correctly (only 0 predictions)
* No surprise, the longer the sequence for the LSTM, the better results,
* Sometimes, for different channels, the predictions on the test set are very similar (look at the picture, channel 7 and 8)
![outputs_and_targets_1](https://github.com/user-attachments/assets/9a342861-fa33-4752-9d06-3a94b13c821b)
The picture is from the test set, the model trained with sequence = 16.

**Comment by kapiblue** [comment]
Two questions:
- which timestep did you predict? The last of the sequence of images or the following one?
- for how many epochs did you train the model?
